src/Tools/dotnet-monitor/LoggingExtensions.cs

```diff
@@ -131,6 +131,18 @@ internal static class LoggingExtensions
                 logLevel: LogLevel.Critical,
                 formatString: "{failure}");
 
+        private static readonly Action<ILogger, Exception> _excessivePermissions =
+            LoggerMessage.Define(
+                eventId: new EventId(21, "ExcessivePermissions"),
```

Comment:
Let's call this "RunningElevated" so as to not imply that running elevated is bad in some way.

Reply:
And rename the field and method to correspond.